src/mdns/mdns_mdnssd.cpp

```diff
@@ -677,13 +678,13 @@ void PublisherMDnsSd::OnServiceResolveFailed(const std::string & aType,
                                              const std::string & aInstanceName,
                                              DNSServiceErrorType aErrorCode)
 {
-    otbrLogWarning("Service %s.%s resolving failed: code=%d", aInstanceName.c_str(), aType.c_str(), aErrorCode);
+    otbrLogWarning("Service %s.%s resolving failed: code=%" PRId32, aInstanceName.c_str(), aType.c_str(), aErrorCode);
```

Comment:
nit: sometimes we start log lines with capital letter, sometimes we do not. Is there a rule for deciding which?

Reply:
I think there's no rule yet. @simonlingoogle @wgtdkp Any idea?

Reply:
OpenThread core starts with capital letter, sounds good to follow.

Reply:
Switched to capital first letter.